bin/i18n/i18n_script_utils.rb

```diff
@@ -168,7 +168,11 @@ def self.get_level_url_key(script, level)
 
   # Used by get_level_from_url, for the script_level-specific case.
   def self.get_script_level(route_params, url)
-    script = Script.get_from_cache(route_params[:script_id])
+    script = if Script.family_names.include?(route_params[:script_id])
```

Comment:
I believe this will be safe, but likely unnecessary. The script that is throwing the error in the PR description, `hello-world-retro`, I believe was deleted after HOC because it was not the script name that curriculum wanted in the end (they wanted it versioned so it became `hello-world-retro-2021`). @davidsbailey can you confirm these scripts were deleted?

So I think the hello world and poetry scripts will need to be cleaned up and this logic should become unnecessary. I'm not sure why you would ever have the family name in the URL on crowdin. We use this logic so that users can go to studio.code.org/s/coursea for instance and be redirected to the latest version of the course. I don't think the i18n sync would want to leverage that feature

Reply:
yes, the unversioned scripts were deleted: https://github.com/code-dot-org/code-dot-org/blob/79ebf6d1fdc09f39421b4218e5da3f73e3f5bff3/dashboard/db/migrate/20220311000924_remove_unused_hoc2021_tutorials.rb#L9